src/components/NFTPage/ControlBar.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <div class="flex items-center">
+    <div v-if="isCollector && view !== 'created'" class="flex gap-[12px]">
+      <ButtonV2
+        size="mini"
+        preset="tertiary"
+        content-class="!text-[12px]"
+        :text="$t('nft_details_page_activity_list_event_transfer')"
+        @click="handleClickTransfer"
+      />
+      <ButtonV2
+        size="mini"
+        preset="tertiary"
+        :is-disabled="true"
+        content-class="!text-[12px]"
+        :text="$t('nft_details_page_button_sell')"
+        @mouseenter.once="handleMouseEnterSell"
+        @click.native="handleClickSell"
+      />
+    </div>
+    <div v-else-if="isWritingNFT && !isCollector || view === 'created'" class="flex gap-[12px]">
```

Comment:
Currently there is an issue with the `isWritingNFT` flag: we only check collection is set in nft class metadata, but if the user failed to submit the nft class to our API, it won't have any price and we cannot collect

Not sure if we want to guard price !== undefined everywhere, or change `this.isWritingNFT` definition in mixin

Reply:
Then guard price in mixin